sn_networking/src/version.rs

```diff
@@ -80,7 +80,7 @@ fn get_truncate_version_str() -> String {
     let version_str = env!("CARGO_PKG_VERSION");
     let parts = version_str.split('.').collect::<Vec<_>>();
     if parts.len() >= 2 {
-        format!("{}.{}", parts[0], parts[1])
+        format!("{}.{}", parts[0], "16")
```

Comment:
Change the version according to #1777

Reply:
This should have been a blocker on that PR. I'll push a commit removing this for `main` integration. (any new network wont need it)